### PR TITLE
Page structure with css

### PR DESCRIPTION
**Pull request by mrnadaara**
Finished styling sign up page. Removed margin properties to allows flexbox to work as intended.
 Added 3 image for the header section to demonstrate services provided.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,65 +1,83 @@
...
+          <p>Already have an Intuit account?</p>
+          <p>
+            <a
+              href="https://accounts.intuit.com/signup.html?offering_id=Intuit.ifs.mint&namespace_id=50000026&redirect_url=https%3A%2F%2Fmint.intuit.com%2Foverview.event%3Futm_medium%3Ddirect%26cta%3Dhero_sign_up_free_ProspectWeb%26adobe_mc%3DMCMID%253D57416333076394992802623129151118719504%257CMCAID%253D2EC0843E05311C99-60000103E0002D07%257CMCORGID%253D969430F0543F253D0A4C98C6%252540AdobeOrg%257CTS%253D1568748821%26ivid%3D773ff057-eb18-4501-9956-ef2354fbfb33#"
+              >I forgot my User ID or Password</a
+            >
+          </p>
+        </section>
+        <button id="signin" class="btn">
+          <i class="fas fa-thumbs-up"></i>
```
Comment (wathigo) [inline index.html]: **[OPTIONAL]** You can remove the thumbs up icon. It's not necessary in this context

File: index.html [hunk 2]
```diff
@@ -1,65 +1,83 @@
...
+        </header>
+        <section id="form-content">
+            <form>
+                <label>
+                    E-mail address
+                    <input type="email" />
+                </label>
+                <label>
+                    Phone
+                    <input type="number" />
```
Comment (wathigo) [inline index.html]: **[REQUIRED]** Use `input` type `tel` for the phone number input field.

File: index.html [hunk 3]
```diff
@@ -1,65 +1,83 @@
...
+                </label>
+                <label>
+                    Password
+                    <input type="password" />
+                </label>
+            </form>
+            <button id="signup" class="btn">
+                <i class="fas fa-thumbs-up"></i>
```
Comment (wathigo) [inline index.html]: **[OPTIONAL]** You can remove the thumbs up icon. It's not necessary in this context

File: index.html [hunk 4]
```diff
@@ -1,65 +1,83 @@
...
-    </body>
-</html>
\ No newline at end of file
+    <main>
+      <header id="content-header">
+        <h3>INTUIT</h3>
+        <ul id="images">
+          <li><img class="logo" src="mint.png" alt="mint" /></li>
+          <li><img class="logo" src="quickbooks.png" alt="mint" /></li>
+          <li><img id="turbo-logo" src="turbotax.png" alt="mint" /></li>
```
Comment (wathigo) [inline index.html]: **[Required]** Use logo on the left and text on the right. You have used an image for both the logo and text. This is why products row is not looking very good.

**Review by wathigo — CHANGES_REQUESTED** [review]
### Hi @Tekcoder  and @mrnadaara ,
**Great work so far :+1: ,**
Just a couple of things to be addressed in order to mark this as complete.

1. We have a problem in styling the form-header element. There should be a small margin between the `h2` and `h3` element. There should be no margin between `h3` and `learn more` link _hint: use span element, remove default margins_
2. We have some errors in the code. Please upload you files to [w3 validator](https://validator.w3.org/#validate_by_upload+with_options) and correct all the errors.

4. You have a lot of commits in the `master` branch. I will let you pass this one for now but make sure that we only have the initial commit in the master branch for your next projects. Please refer to [github flow](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/) for more info.

3. Check all the comments below and implement the required changes.

**Please do the necessary changes then submit another request for review!**

**Review by Redvanisation — APPROVED** [review]
### Hello @mrnadaara 

**_Good job you have passed all the previous requirements, I will approve the project but you have to solve the conflict in the branch when you merge to master._**

### Good luck in the next project! 👍

**Pull request merged**